features/FxA-80-eye-password-experiment/README.md

```diff
@@ -0,0 +1,119 @@
+
+Eye Password Experiment
+========================
+
+https://mozilla.aha.io/features/FXA-80
+
+Firefox Account users are struggling immensely when changing their passwords.
+
+![Change Password Statistics](change-password-stats.png)
+
+[View Live Chart](https://app.datadoghq.com/graph/embed?token=f39f21f7fb65ebbd224d49e88dd365b817e869d68d2a9e384f56e908e0df666b&height=500&width=1000&legend=true)
+
+We would like to learn how we can help them struggle less by experimenting with different password unmasking treatments. We may use these results to inform a choice about unmasking passwords in platform.
+
+# Control
+Currently we use a simple “show” toggle. Both password field toggles are synchronized which we should remove this for the experiment (and permanently as filed in [#3600](https://github.com/mozilla/fxa-content-server/issues/3600)).
+
+<img alt="Current Masked" src="current-masked.png" width="525">
+
+<img alt="Current Unmasked" src="current-unmasked.png" width="525">
+
+# Treatments
+## Match 1 - Control vs. No Show
+
+We believe that removing the unmasking functionality from both new and old password fields for people changing their passwords will achieve more errors and bounces. We will know this is true when we see an increase in bounces to the section, and Task Success errors outlined below.
+
+<img alt="No Show" src="eye-password-noshow.png" width="525">
+
+## Match 2 - Control vs. Checkbox
+
+We believe that adding a checkbox to the Show control for users changing their password
+will achieve greater engagement. We will know this is true when we see this greater amounts of unmasking, entering a password, optionally hiding and then submitting the form.
+
+<img alt="Checkbox Masked" src="eye-password-checkbox1.png" width="380">
+
+<img alt="Checkbox Unmasked" src="eye-password-checkbox2.png" width="380">
+
+## Match 3 - Checkbox vs. Eyeball
+
+We believe that using an Eye icon with a tooltip to the Show control for users changing their password will achieve greater engagement. We will know this is true when we see this greater amounts of unmasking, entering a password, optionally hiding and then submitting the form.
+
+<img alt="Eyeball Masked" src="eye-password-eyeball1.png" width="380">
+
+<img alt="Eyeball Unmasked" src="eye-password-eyeball2.png" width="380">
+
+The eye color matches our current smokey grey and blue colors. We should use use a tooltip so that users know what the eye does. The deprecated eye in animation for position only.
+
+<img alt="Toggle" src="eye-password-toggle.gif" width="525">
+
+### Acceptance Criteria
+
+* When the user hovers over the control and the password is masked, there should be a tooltip that says "Show password"
```

Comment:
Not sure if it is worthwhile to mention how this functions on mobile devices. Since they don't have hover states.